server/api.go

```diff
@@ -625,53 +631,136 @@ func (p *Plugin) createIssue(c *UserContext, w http.ResponseWriter, r *http.Requ
 	p.writeAPIResponse(w, result)
 }
 
-func (p *Plugin) getPermaLink(postID string) string {
+func (p *Plugin) attachCommentToIssue(c *UserContext, w http.ResponseWriter, r *http.Request) {
+	var issue *gitlab.IssueRequest
+
+	if err := json.NewDecoder(r.Body).Decode(&issue); err != nil {
+		c.Log.WithError(err).Warnf("error decoding JSON body to attach comment to the issue")
+		p.writeAPIError(w, &APIErrorResponse{ID: "", Message: fmt.Sprintf("error decoding JSON body to attach comment to the issue. Error: %s", err.Error()), StatusCode: http.StatusBadRequest})
```

Comment:
```suggestion
		p.writeAPIError(w, &APIErrorResponse{ID: "", Message: fmt.Sprintf("error decoding JSON body for attach comment to the issue. Error: %s", err.Error()), StatusCode: http.StatusBadRequest})
```
You can create a function which prints an error in server logs and also writes the error in the response and call that function here.

Reply:
discussed